lib/oboeRequest.js

```diff
@@ -23,13 +23,14 @@ class OboeRequest {
    * Make a streaming query to the Devo API.
    *
    * @param {Object} options An object with method, body and headers.
+   * @param {Object} processParams An object with stream process opt-in parameters
```

Comment:
Isn't it better to pass the options to the constructor `OboeRequest()` and store as a private attribute `_parseMetadata`?

Reply:
Yes, it's done.

Reply:
I closed the js-helper PR.

Reply:
Please [document the new option](https://github.com/DevoInc/browser-sdk#query-options), and update version accordingly. Thanks! :)

Reply:
Ready. Checkout please.

Reply:
You still need to update the version.

Reply:
Ready, changed to 1.2.6